starknet_py/net/client_models.py

```diff
@@ -295,13 +295,20 @@ class DeclaredContractHash:
     compiled_class_hash: int
 
 
+@dataclass
+class ReplacedClass:
+    contract_address: int
+    class_hash: int
+
+
 @dataclass
 class StateDiff:
     deployed_contracts: List[DeployedContract]
     deprecated_declared_contract_hashes: List[int]
     declared_contract_hashes: List[DeclaredContractHash]
     storage_diffs: List[StorageDiffItem]
     nonces: List[ContractsNonce]
+    replaced_classes: List[ReplacedClass]
```

Comment:
Adding this as non-optional makes the current version of library not compliant with any rpc spec version.
Do we plan to support rpc spec v0.3.0 in the next release?

Reply:
Making this non-optional is not problematic because we can just provide empty list if `replaced_classes` are not present in the response. And it will be added to the RPC spec anyway https://github.com/starkware-libs/starknet-specs/pull/76